proto/process/agent.proto

```diff
@@ -1131,6 +1132,26 @@ message ContainerStatus {
 	string imageID = 8;
 }
 
+message NodeAffinity {
+  NodeSelector requiredDuringSchedulingIgnoredDuringExecution = 1;
+  repeated PreferredSchedulingTerm preferredDuringSchedulingIgnoredDuringExecution = 2;
+}
+
+message NodeSelector {
+  repeated NodeSelectorTerm nodeSelectorTerms = 1;
+}
+
+message NodeSelectorRequirement {
```

Comment:
where do we use this one ?

Reply:
We don't! It was a leftover from an earlier version.